templates/deployment-sidekiq.yaml

```diff
@@ -200,6 +201,18 @@ spec:
           {{- with $context.Values.volumeMounts }}
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- if $context.Values.mastodon.sidekiq.readinessProbe.enabled }}
+          readinessProbe:
+            failureThreshold: {{ default 10 $context.Values.mastodon.sidekiq.readinessProbe.failureThreshold }}
+            exec:
+              command:
+              - cat
+              - /var/www/tmp/sidekiq_process_has_started_and_will_begin_processing_jobs
```

Comment:
I think the path is `/opt/mastodon/tmp/sidekiq_process_has_started_and_will_begin_processing_jobs` in our containers?

Reply:
Also https://github.com/mastodon/mastodon/pull/30988 allows people to change the filename for this. I do not know why someone would do it, but it will become possible. Our default Dockerfile sets it to the name above.